### PR TITLE
updating syntax to match julia 0.4.0

### PR DESCRIPTION
**Pull request by vshesh**
Thank you for suggesting this package!

### CODE REVIEW

File: src/MacroExpandJL.jl [hunk 1]
```diff
@@ -158,7 +158,7 @@ function show_block(io::IO, head, args::Vector, body, indent::Int)
         end
         exs = body.args
     else
-        exs = {body}
+        exs = [body]
```
Comment (timholy) [inline src/MacroExpandJL.jl]: Should probably be `Any[body]` wherever you're replacing `{}`.

**Comment by timholy** [comment]
Thanks for contributing an update! Are you sufficiently git-comfortable to push a commit that addresses the tiny issue I flagged? If not, I can fix it (or coach you through it).

**Comment by timholy** [comment]
Oh, and we should probably add a `REQUIRE` file with contents `julia 0.4`.

**Comment by vshesh** [comment]
Ok, those issues should be fixed.

**Comment by timholy** [comment]
Thanks so much!

**Pull request merged**